### PR TITLE
remove Mydumper and tikv-importer

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Starting from v7.5.0, Mydumper and tikv-importer are deprecated.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-error-handling.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ However, you need to reset the data migration task in some cases. For details, r
...
-| `code=32001` |   Abnormal dump processing unit                                            |  If the error message contains `mydumper: argument list too long.`, configure the table to be exported by manually adding the `--regex` regular expression in the Mydumper argument `extra-args` in the `task.yaml` file according to the block-allow list. For example, to export all tables named `hello`, add `--regex '.*\\.hello$'`; to export all tables, add `--regex '.*'`. |
+| `code=32001` |   Abnormal dump processing unit                                            |  If the error message contains `mydumper: argument list too long.`, configure the table to be exported by manually adding the `--regex` regular expression in the `mydumpers.extra-args` argument in the `task.yaml` file according to the block-allow list. For example, to export all tables named `hello`, add `--regex '.*\\.hello$'`; to export all tables, add `--regex '.*'`. |
```
Comment (Oreoxmt) [inline dm/dm-error-handling.md:108]: @Frank945946 PTAL

File: dumpling-overview.md [hunk 2]
```diff
@@ -46,11 +46,9 @@ TiDB also provides other tools that you can choose to use as needed.
 
 > **Note:**
 >
-> PingCAP previously maintained a fork of the [mydumper project](https://github.com/maxbube/mydumper) with enhancements specific to TiDB. This fork has since been replaced by [Dumpling](/dumpling-overview.md), which has been rewritten in Go, and supports more optimizations that are specific to TiDB. It is strongly recommended that you use Dumpling instead of mydumper.
->
-> For more information on Mydumper, refer to [v4.0 Mydumper documentation](https://docs.pingcap.com/tidb/v4.0/backup-and-restore-using-mydumper-lightning).
+> PingCAP previously maintained a fork of the [mydumper project](https://github.com/maxbube/mydumper) with enhancements specific to TiDB. Starting from v7.5.0, [Mydumper](https://docs.pingcap.com/tidb/v4.0/mydumper-overview) is deprecated and most of its features have been replaced by [Dumpling](/dumpling-overview.md). It is strongly recommended that you use Dumpling instead of mydumper.
```
Comment (Oreoxmt) [inline dumpling-overview.md:49]: @Frank945946 PTAL

File: ecosystem-tool-user-guide.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ The following are the basics of Dumpling:
 
 > **Note:**
 >
-> PingCAP previously maintained a fork of the [mydumper project](https://github.com/maxbube/mydumper) with enhancements specific to TiDB. This fork has since been replaced by [Dumpling](/dumpling-overview.md), which has been rewritten in Golang, and provides more optimizations specific to TiDB. It is strongly recommended that you use Dumpling instead of mydumper.
+> PingCAP previously maintained a fork of the [mydumper project](https://github.com/maxbube/mydumper) with enhancements specific to TiDB. Starting from v7.5.0, [Mydumper](https://docs.pingcap.com/tidb/v4.0/mydumper-overview) is deprecated and most of its features have been replaced by [Dumpling](/dumpling-overview.md). It is strongly recommended that you use Dumpling instead of mydumper.
```
Comment (Oreoxmt) [inline ecosystem-tool-user-guide.md:78]: @Frank945946 PTAL

File: dumpling-overview.md (hunk 2 above)
Comment (Frank945946) [reply dumpling-overview.md:49]: LGTM

File: ecosystem-tool-user-guide.md (hunk 3 above)
Comment (Frank945946) [reply ecosystem-tool-user-guide.md:78]: LGTM

File: tidb-cloud/changefeed-sink-to-mysql.md [hunk 4]
```diff
@@ -69,7 +69,7 @@ To load the existing data:
     SET GLOBAL tidb_gc_life_time = '720h';
     ```
 
-2. Use [Dumpling](https://docs.pingcap.com/tidb/stable/dumpling-overview) to export data from your TiDB cluster, then use community tools such as [mydumper/myloader](https://centminmod.com/mydumper.html) to load data to the MySQL service.
+2. Use [Dumpling](https://docs.pingcap.com/tidb/stable/dumpling-overview) to export data from your TiDB cluster, then use community tools such as Dumpling to load data to the MySQL service.
```
Comment (Frank945946) [inline tidb-cloud/changefeed-sink-to-mysql.md:72]:
```suggestion
2. Use [Dumpling](https://docs.pingcap.com/tidb/stable/dumpling-overview) to export data from your TiDB cluster, then use community tools such as myloader to load data to the MySQL service.
```

File: tidb-lightning/tidb-lightning-configuration.md [hunk 5]
```diff
@@ -54,9 +54,6 @@ enable-diagnose-logs = false
 # Each table is split into one "index engine" to store indices, and multiple
 # "data engines" to store row data. These settings control the maximum
 # concurrent number for each type of engines.
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-configuration.md:56]:
```suggestion
# concurrent number for each type of engines. You can typically use the following two default values.
```

File: dm/dm-error-handling.md (hunk 1 above)
Comment (Frank945946) [reply dm/dm-error-handling.md:108]: 不需要改，这里是 DM 里面内置的 Mydumper

**Review by D3Hunter — APPROVED** [review]
tikv-importer part lgtm

**Review by lichunzhu — APPROVED** [review]
Mydumper part LGTM

**Review by ran-huang — APPROVED** [review]

File: tidb-lightning/troubleshoot-tidb-lightning.md [hunk 6]
```diff
@@ -119,23 +119,6 @@ tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-error-destroy=
...
-
-    Max Memory Usage ≈ `max-open-engines` × `write-buffer-size` × `max-write-buffer-number`
-
-2. Decrease the value of `table-concurrency` + `index-concurrency` so it is less than `max-open-engines`.
-
-3. Restart `tikv-importer` to forcefully remove all engine files (default to `./data.import/`). This also removes all partially imported tables, which requires TiDB Lightning to clear the outdated checkpoints.
-
-    ```sh
-    tidb-lightning-ctl --config conf/tidb-lightning.toml --checkpoint-error-destroy=all
-    ```
```
Comment (Oreoxmt) [inline tidb-lightning/troubleshoot-tidb-lightning.md:122]:
```suggestion
```

**Comment by Oreoxmt** [comment]
Close by https://github.com/pingcap/docs/pull/15289

**Pull request closed without merging**